student/src/test/java/tds/student/sql/repository/RemoteExamSegmentRepositoryTest.java

```diff
@@ -17,6 +17,8 @@
 import java.net.URI;
 import java.util.UUID;
 
+import tds.student.sql.repository.remote.impl.RemoteExamSegmentRepository;
```

Comment:
This import actually is in use on line 29 of this same file.  I should've called this out as a symptom of moving some of the files around.